lib/codegen.js

```diff
@@ -53,6 +53,11 @@ var getViewForSwagger2 = function(opts, type){
             };
             op.parameters = op.parameters ? op.parameters : [];
             op.parameters.forEach(function(parameter) {
+                if ('$ref' in parameter) { 
```

Comment:
Can you provide a sample swagger api that uses this feature?
I'm not familiar with the the "in" syntax, is it safe to use? Why is there two if instead of one?